globals.go

```diff
@@ -581,8 +581,9 @@ var (
 	}
 
 	// constant arrays for db update logic
-	AbortableStatuses = []string{TaskStarted, TaskDispatched}
-	CompletedStatuses = []string{TaskSucceeded, TaskFailed}
+	AbortableStatuses   = []string{TaskStarted, TaskDispatched}
+	CompletedStatuses   = []string{TaskSucceeded, TaskFailed}
+	UncompletedStatuses = []string{TaskStarted, TaskUnstarted, TaskUndispatched, TaskDispatched, TaskConflict, TaskInactive}
```

Comment:
TaskInactive doesn't seem like an uncompleted status, because we don't want it to ever run, right?